web/src/shared/components/Header/Search.tsx

```diff
@@ -80,7 +81,7 @@ const resultRenderer = ({ ...props }) => {
         available = `No access to this ${props.type}`
         colour = 'gray'
     }
-    if (props.type === 'participant' || props.type === 'family') {
+    if (props.type === 'participant') {
```

Comment:
Can we type this component, and formalise the type param?

Reply:
Added the enum for the type param